packages/core/src/message-provider.ts

```diff
@@ -135,20 +135,21 @@ export function createMessageProvider({
           captureException(err);
         }
       }
-  
+
       for (const [did, messageIds] of Object.entries(messageIdsPerDid)) {
         logger.debug(`Acknowledging messages for ${did}`);
         let startTime = new Date().getTime();
-        // FIXME: https://dock-team.atlassian.net/browse/DCKM-415
-        // We don't need to wait for the ack to complete
-        // Those messages were already stored in the wallet
-        // I will let this as is for now until we see the full performance report
-        await relayService.ackMessages({
-          did,
-          messageIds,
-        });
-
-        logger.performance('Acknowledged messages', startTime);
+        relayService
+          .ackMessages({
+            did,
+            messageIds,
+          })
+          .catch(err => {
+            console.error('Failed to ack messages', err.message);
+          })
+          .finally(() => {
+            logger.performance('Acknowledged messages', startTime);
```

Comment:
this feels like it would be confusing because in the case of an error the log will show:

```
Failed to ack messages
Acknowledged messages
```

Reply:
good point, I think we should log that only if it succeed then

Reply:
i've pushed an update